manager/install.sh

```diff
@@ -38,7 +38,7 @@ function ensure_eks() {
     echo -e "￮ spinning up the cluster ... (this will take about 15 minutes)\n"
 
     python generate_eks.py $CORTEX_CLUSTER_CONFIG_FILE > $CORTEX_CLUSTER_WORKSPACE/eks.yaml
-    eksctl create cluster --timeout=$EKSCTL_TIMEOUT -f $CORTEX_CLUSTER_WORKSPACE/eks.yaml
+    eksctl create cluster --timeout=$EKSCTL_TIMEOUT --install-neuron-plugin=false -f $CORTEX_CLUSTER_WORKSPACE/eks.yaml
```

Comment:
Yes; unless you add this flag, the new version of eksctl automatically installs the device plugin for you when you create the cluster (if you are using inf instances). But we need to have our own config for this, since we add some things like resource requests, tolerations, etc. We are also using a newer version than they use

Reply:
I see. Got it.